internal/config/profile.go

```diff
@@ -243,6 +244,10 @@ func (p *Profile) Name() string {
 
 func SetName(name string) { Default().SetName(name) }
 func (p *Profile) SetName(name string) {
+	if strings.Contains(name, ".") {
+		log.Fatal("Profile should not contain '.'")
```

Comment:
the CLI is arch in (broadly terms) 3 layers,`cli`, handles inputs/outputs, `store` communication with the backend and `config` the cl configuration
using logs here is not ideal, ideal solution your `Set` name should now return an error but that may be a lot of small changes, another option is before hitting `Save` of the config to validate the profile, or validate the input

TL;DR leave console outputs to the CLI layer use error handling properly